### PR TITLE
Fix the GSS-Proxy AD integration procedure

### PR DESCRIPTION
**Pull request by adamlazik1**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

**Comment by adamlazik1** [comment]
According to @alsouza, setting the SPN might not be necessary at all. To be confirmed by retesting next week.

**Comment by adamruzicka** [comment]
> To be confirmed by retesting next week.

Did this happen?

**Comment by adamlazik1** [comment]
Not yet, but hopefully on the way. I'll ask if there has been any progress.

File: guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc [hunk 1]
```diff
@@ -47,17 +47,43 @@ Apache user must not have access to the keytab file.
...
+----
+// TO BE CONFIRMED: This step should not be needed
+. Add a *Service Principal Name (SPN)* for your AD server.
+You can use a tool such as `adcli`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# adcli spn --add _My_Service_/{foreman-example-com} _My_Username_
+----
+. Make sure the following line is included in the `/etc/sssd/sssd.conf` file:
```
Comment (asteflova) [inline guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc:76]:
```suggestion
. Configure SSSD to map Group Policy Objects (GPOs) from AD to the `foreman` PAM service by ensuring the following line is included in the `/etc/sssd/sssd.conf` file:
```

I don't know if this wording is technically correct, but my point is to suggest to describe what adding the line does.

File: guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc [hunk 2]
```diff
@@ -47,17 +47,43 @@ Apache user must not have access to the keytab file.
...
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+ad_gpo_map_service = +foreman
+----
+. If you changed the `/etc/sssd/sssd.conf` configuration file, restart the sssd service:
```
Comment (asteflova) [inline guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc:82]:
```suggestion
. If you changed the `/etc/sssd/sssd.conf` configuration file, restart the `sssd` service:
```
It's either "the SSSD service" or "the `sssd` service". And other steps reference services in the format of "the `httpd` service" etc.

File: guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc (hunk 1 above)
Comment (asteflova) [reply guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc:76]: Also, the IMPORTANT box right under this procedure also tells users to add this config option to sssd.conf. That could be confusing. Do you have any ideas for how to deal with it? I think you could take the entire IMPORTANT box and merge it with this step... would that make sense?

**Comment by ekohl** [comment]
Typo in the commit message: GGS -> GSS

**Comment by asteflova** [comment]
@adamlazik1 and I agreed that I would take over his effort to update and verify the procedure. I filed https://github.com/theforeman/foreman-documentation/pull/3056 for that purpose.

**Pull request closed without merging**